user_guide/intro_nav_pinned_pages.rst

```diff
@@ -0,0 +1,61 @@
+.. _user-guide-ui-components-pinned:
+
+Pinbar
+======
+
+Pinning pages is a convenient tool that allows you to save a page and easily return to it. When you open a pinned page, 
+it will look exactly as it did when you have pinned it.
```

Comment:
Edit: Pinning pages is a convenient tool that allows you to save a page and easily return to it. When you open a pinned page, it will look exactly as it did when you first pinned it.